src/methods.jl

```diff
@@ -329,3 +329,52 @@ function combine(frames::Vararg{<:AbstractArray}; method = median)
 end
 
 combine(frames::AbstractVector{<:AbstractArray}; method = median) = combine(frames..., method = method)
+
+
+"""
+    subtract_dark!(frame::AbstractArray, dark_frame::AbstractArray; data_exposure = 1, dark_exposure = 1, scale = false)
+
+In place version of [`subtract_dark`](@ref)
+
+# See Also
+[`subtract_dark`](@ref)
+"""
+function subtract_dark!(frame::AbstractArray, dark_frame::AbstractArray; data_exposure = 1, dark_exposure = 1, scale = false)
+    if scale
+        frame .-= (dark_frame .* (data_exposure / dark_exposure))
```

Comment:
Why isn't this broadcasting everything?  You're creating a temporary array.  Doesn't
```suggestion
        @. frame -= (dark_frame * (data_exposure / dark_exposure))
```
work as expected?

Also, is the `scale` argument necessary?  I think you can just always this line.  If one wants to scale has to remember to also to set `scale = true` in addition to setting the exposure, it's a bad gotcha

Reply:
On second look yes having scale be an argument seems to be a remnant of ancient iraf code. I agree it would be better to skip it.

Reply:
Sure, I will make the changes!